ui-src/src/app/configurations/configurations.component.html

```diff
@@ -1,21 +1,21 @@
 <div class="container">
     <div class="row">
-        <div class=" column  col-md-7">
+        <div class=" column  col-md-8">
             <label>Config File Path *</label>
             <input class="form-control" type="string" [(ngModel)]="configPath" id="config-path" required>
+
         </div>
         <div class=" column col-md-4">
-            <label>Xml File Name*</label>
-            <input class="form-control" type="string" [(ngModel)]="xmlFileName" id="xml-file-name" required>
+            <label>File Select</label>
+            <input class="form-control" placeholder="Click to Browse Files" [(ngModel)]="xmlFileName" type="string" onclick="javascript:document.getElementById('file').click();">
+            <input id="file" type="file" style='visibility: hidden;' (change)="fileSelector($event)" />
```

Comment:
This style should be in class, it is bad practice to do have styles like this in the html like this. Also if you are trying to hide that input, it might be better to remove it from the dom by using *ngIf